### ISSUE TITLE
Insufficient Speech entering finger numbers, lyrics, chords, etc

### ISSUE THREAD

**Issue by jmwmusic** [musescore/MuseScore#34538]
### Issue type

Accessibility issue (e.g. for keyboard-only or screen reader users)

### Description with steps to reproduce

Currently when you are entering Lyrics, Finger Numbers, Chords, Roman Numerals (and possibly other items) when you advance to a new note you hear 
"Finger Number bar X, Beat X, edit blank".

This means for a blind user you have to try and memorise the notes of your score and imagine print or Braille as you move to know what note you are adding these to.  This means most users could only enter a couple at a time without having to come out of that mode and check their position, memorise the next few notes and resume.

Instead it should speak the pitch name and other details as usual when you navigate through a score.  Some mid note positions for Chords would best still just speak the bar and beat as it does currently.  

It would also be helpful if the notes could play as usual when navigated on to.
 

### Supporting files, videos and screenshots

N/A

### In which versions of MuseScore Studio is this issue present?

4.7.4

### Regression

I was unable to check

### Operating system

Windows 11 with NVDA

### Checklist

- [x] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [x] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [x] I have attached all requested files and information to this report
- [x] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible
